reset.sh

```diff
@@ -25,7 +25,7 @@ confirm "☠️  Warning! 😳 This is highly destructive! 😱 Are you sure you
 echo "Okay ... good luck! 😰"
 
 # Hit the reset button.
-docker compose down --volumes
+docker compose down --volumes --rmi local
```

Comment:
This is not a hard requirement but having `--remove-orphans` would help, when we add/remove services to the `docker-compose.yml` file. After a checkout, they will become "orphaned" by this terminology and unless that option is passed, they would stay running. This is not something we want ~99% of the time.

Reply:
Sorry, I missed https://github.com/getsentry/onpremise/pull/988#discussion_r648240411 earlier. Done in e405249807bd4d13bc09ef3a3025adb03e651830. 👍